genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
```

Comment:
Good work implementing the add genre method, but this is maybe a little confusing. Let me explain how this should work in the application

- When you add a music album, the application should ask you for the name of the genre
- Once the genre is added, you list the music albums. It should print the name of the music albums with the name of the genre. Something like `Music album name, genre name of the genre`.